locales/de.po

```diff
@@ -0,0 +1,136 @@
+# #-#-#-#-#  metabase-backend.pot (metabase)  #-#-#-#-#
+# German translations for metabase package.
+# Copyright (C) 2017 Metabase <docs@metabase.com>
+# This file is distributed under the same license as the metabase package.
+# Tom Robinson <tom@metabase.com>, 2017.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: metabase\n"
+"Report-Msgid-Bugs-To: docs@metabase.com\n"
+"POT-Creation-Date: 2017-08-21 21:54+0200\n"
+"PO-Revision-Date: 2017-08-21 21:17+0200\n"
+"Last-Translator: Tom Robinson <tom@metabase.com>\n"
+"Language-Team: German <translation-team-de@lists.sourceforge.net>\n"
+"Language: de\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"#-#-#-#-#  metabase-frontend.pot  #-#-#-#-#\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"#-#-#-#-#  metabase-backend.pot (metabase)  #-#-#-#-#\n"
+
+#: frontend/src/metabase/home/components/Activity.jsx:131
+msgid "Hello World!"
+msgstr "Hallo Welt!"
+
+#: frontend/src/metabase/home/components/Activity.jsx:132
```

Comment:
does it make sense to include all these line numbers? They'll surely get out of date pretty quickly

Reply:
That's why in my opinion it's important to get these updates happening out of our normal dev flow. AFAIK having the line numbers is required for the translation software we're using. Any change to a file that changes the line number of a translation, needs to have this updated. Because of that, there's the potential for a lot of churn on this file. Also since it's a separate thing thats needs to be ran, we're pretty likely to forget it.